dace/sdfg/state.py

```diff
@@ -298,10 +298,11 @@ def scope_tree(self) -> 'dace.sdfg.scope.ScopeTree':
 
         # Get scopes
         for node, scopenodes in sdc.items():
+            scope_exit_nodes = [v for v in scopenodes if isinstance(v, nd.ExitNode)]
```

Comment:
Does this make any meaningful difference, or is it for debugging? If it is only for debugging, I would suggest, after completing the PR, switching it to how it was before because it is somewhat slower.

Reply:
Was for debugging and switched it back